<launch>
  <node name="acpi_acu" pkg="action_control_unit" type="api_acu.py" respawn="true"/>
  <node name="direction" pkg="action_control_unit" type="direction.py" respawn="true"/>
  <node name="custom_to_ackermann" pkg="action_control_unit" type="custom_to_ackermann.py" respawn="true"/>
  <node name="ackermann_steering" pkg="action_control_unit" type="ackermann_steering.py" respawn="true"/>
</launch>